components/header/index.styl

```diff
@@ -1,77 +1,106 @@
+header
+  height 50vh
+  position relative
+
+.slide-show-container
+  position relative
+  height 100%
+
+.header-slide
+  width 100%
+  height 100%
+  background-position center center
+  background-size cover
+  -webkit-transition: opacity 1s ease-in
+  -moz-transition opacity 1s ease-in
+  -o-transition opacity 1s ease-in
+  -ms-transition opacity 1s ease-in
+  transition opacity 1s ease-in
+  position absolute
+  top 0
+  background-image url('./images/banner-3.jpg')
+  opacity 0
+
+.active-banner
+  opacity 1
+  z-index 500
+
+.slide-1
+  background-image url('./images/banner-1.jpg')
+
+.slide-2
+  background-image url('./images/banner-2.jpg')
+
+.slide-3
+  background-image url('./images/banner-3.jpg')
+
+.header-content
+  height 100%
+  display flex
+  flex-direction column
+  justify-content space-around
+  z-index 501
+  position absolute
+  top 0
+
+.header-text
+  width 85%
+  margin 10% auto
+  padding-top 30px !important
+  color white
+  avant-garde-size('s-headline')
+  font-size 20px
+  > p
+    margin 20px 0
+    &:first-child
+      margin-top 0
+    &:last-child
+      margin-bottom 0
+
+.header-section-nav
+  display none
+
 // mobile
 @media (max-width 320px)
   .header-text > p
-    font-size 15px
+    font-size 13px
 
-@media (max-width 480px)
+@media (orientation: landscape)
```

Comment:
This is the reason you had to override the header styling with `!important`, you can apply multiple properties to media queries to specifically target phones in landscape mode. For example:
```
@media only screen 
  and (min-device-width: 320px) 
  and (max-device-width: 568px)
  and (-webkit-min-device-pixel-ratio: 2)
  and (orientation: landscape)
```